extensions/scheduler/deployment/src/test/java/io/quarkus/scheduler/test/DisabledSchedulerTest.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.scheduler.test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import javax.inject.Inject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.Scheduler;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class DisabledSchedulerTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(Jobs.class)
+                    .addAsResource(new StringAsset("quarkus.scheduler.enabled=false"),
+                            "application.properties"));
+
+    @Inject
+    Scheduler scheduler;
+
+    @Test
+    public void testNoSchedulerInvocations() throws InterruptedException {
+        assertFalse(scheduler.isRunning());
+    }
+
+    static class Jobs {
```

Comment:
Since we are only testing if scheduler is running or not, I think we can live off this class a.k.a we do not need it.

Reply:
I think that we do need it because if there is no scheduled method then `SimpleScheduler` is never started.